sklearn/linear_model/coordinate_descent.py

```diff
@@ -464,10 +496,11 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
             model = cd_fast.enet_coordinate_descent_gram(
                 coef_, l1_reg, l2_reg, precompute, Xy, y, max_iter,
                 tol, rng, random, positive)
+            # XXX screening is not performed in gram/precomputed case
```

Comment:
In fact I have worked on another unfinished PR for that. 
But I prefer merge this one alone.